app/assets/stylesheets/components/maps/_v-map-baselayer-controls.scss

```diff
@@ -3,35 +3,48 @@
 }
 .v-map-baselayer-controls {
   position: absolute;
-  right: rem-calc(46);
-  bottom: rem-calc(32);
+  bottom: rem-calc(10);
+  right: rem-calc(10);
 
-  display: flex;
-  align-content: flex-end;
+  @include breakpoint($small) {
+    bottom: rem-calc(16);
+  }
+
+  @include breakpoint($medium) {
+    right: rem-calc(16);
+  }
 
   &__control {
     min-width: rem-calc(100);
 
-    border: 1px solid darken($grey-xlight, 5%);
+    background-color: $white;
+    border: rem-calc(1px) solid $grey-dark;
     border-radius: 0;
```

Comment:
I know you know this (I'm guilty of it too!) don't need to have the px here (and good point, I've forgotten to apply rem-calc to all my borders)

Reply:
Oops yep, will take out